argo/common/run_completion_event_unit_test.go

```diff
@@ -0,0 +1,189 @@
+//go:build unit
+
+package common
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Context("RunCompletionEvent.Validate", func() {
+
+	validate, _ = InitialiseValidation()
+
+	namespaceName := NamespacedName{
+		Name:      "Name",
+		Namespace: "Namespace",
+	}
+
+	baseEvent := RunCompletionEvent{
```

Comment:
It is weird to me that the baseEvent is an invalid one... usually that would be the valid one and then you make it invalid in those tests?

Reply:
its because they are both set, we could make them both empty instead, which we will do now

Reply:
changed in [e78e58f](https://github.com/sky-uk/kfp-operator/pull/352/commits/e78e58f3ef3c6464dd3d9c2fcce26544d39f5512)